bom/pom.xml

```diff
@@ -27,6 +27,7 @@
         <infinispanVersion>8.0.2.Final</infinispanVersion>
         <mongodbVersion>3.0.3</mongodbVersion>
         <fongodbVersion>2.0.2</fongodbVersion>
+        <orientdbVersion>[2.1.9,)</orientdbVersion>
```

Comment:
We don't use range for versions, it migh tlead to some unpredictable results.
Please, change it to a fixed version.